src/main/webapp/app/assessment/assessment-header/assessment-header.component.html

```diff
@@ -31,6 +31,7 @@ <h3 class="top-container flex-wrap flex-lg-nowrap">
             id="moreFeedbackRequest"
             *ngIf="
                 isAssessor &&
+                exercise &&
                 !exercise?.isAtLeastInstructor &&
```

Comment:
Now the questionmark is no longer needed here.
```suggestion
                !exercise.isAtLeastInstructor &&
```